routers/web/repo/issue.go

```diff
@@ -291,6 +291,8 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption uti
 		return
 	}
 
+	ctx.Data["Posters"] = ctx.Data["Assignees"]
```

Comment:
I don't think this is right; I believe the list of users that can open a PR is not the same as the list of possible assignees, but I can't find an existing method to get the list of users that can open a PR. Is this available? If not, does anyone have pointers on how I could write this method, similar to GetRepoAssignees()?

Reply:
Yeah this is very wrong, to provide some pointers on how to create a new functions for this(there currently isn't one). 

You want to craft a function that is along the lines of:
```go
	userIDs := make([]int64, 0, 8)
	return userIDs, db.GetEngine(ctx).Table("issue").
		Cols("poster_id").
		Where("is_pull = ?", true).
		And("repo_id = ?", repoID).
		Distinct("poster_id").
		Find(&userIDs)
```
Which would give you the list of all posters of a pull request in a given repo. Not sure if it works, but it should be something along the lines of that.

Reply:
Great, thanks for the snippet! I'll work on this this weekend.

Reply:
Done in 23be14b0cca94c0ea64a12224f1eeca48506a2eb